cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -658,12 +658,13 @@ installExes verbosity baseCtx buildCtx platform compiler
 
       mkFinalExeName :: UnqualComponentName -> FilePath
       mkFinalExeName exe = prefix <> unUnqualComponentName exe <> suffix <.> exeExtension platform
-      installdirUnknown =
+      installdirUnknown path =
```

Comment:
`installPath` is in scope, so doesn't need to be passed as an argument? does it?

Reply:
You're completely right, updated.